src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/helpers/HmrcLayout.scala.html

```diff
@@ -70,7 +72,7 @@
       additionalHeadBlock = additionalHeadBlock,
       additionalScriptsBlock = additionalScriptsBlock,
       beforeContentBlock = beforeContentBlock,
-      mainContentLayout = mainContentLayout,
+      mainContentLayout = mainContentLayout.orElse(Some(noLayoutWrapper)),
       pageLayout = pageLayout,
```

Comment:
I think [pageLayout has a similar change in semantics](https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukTemplate.scala.html#L90) so we might want to change that too

Reply:
ah, good spot, will update

Reply:
have updated to restore the previous behaviour with `pageLayout`, but because this function is `PageLayout => Html` it's ended up tightly coupled to the implementation of `GovukTemplate` - not sure if there's a better way to do this.

Reply:
can't think of anything, I think in practice it won't change much so low cost I reckon